internal/command/ls_runs.go

```diff
@@ -28,7 +29,9 @@ baur ls runs -s duration-desc calc               list task runs of the calc
 						 application, sorted by
 						 run duration
 baur ls runs --csv --after=2018.09.27-11:30 '*'  list all task runs in csv format that
-						 were started after 2018.09.27 11:30`
+						 were started after 2018.09.27 11:30
+baur ls runs --limit=1 calc                      list a single task run of the calc
```

Comment:
indentation is off here

Reply:
It appears that spaces instead of tabs have been used on the lines that begin with `baur ls`, I'll fix this up for all of the examples in this file.